src/zustand/store.js

```diff
@@ -2,6 +2,16 @@ import { create as createZustand } from "zustand";
 
 const useStore = createZustand((set) => ({
```

Comment:
I think it is more common to call this `createStore` but it's not super important. :)